src/app/pages/results-summary/results-summary.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-results-summary',
+  templateUrl: './results-summary.component.html',
+  styleUrls: ['./results-summary.component.scss']
+})
+export class ResultsSummaryComponent implements OnInit {
+  constructor(private router: Router) {
+  }
+
+  ngOnInit(): void {
+    // eslint-disable-next-line no-undef
```

Comment:
Is this rule disabled in order to safely reference `sessionStorage`? If not, I don't quite see why this rule is disabled here.

Reply:
That is correct. The linter doesn't allow sessionStorage without error due to undefined.

Reply:
Ok, that's fine then.